packages/@aws-cdk/aws-stepfunctions/lib/states/map.ts

```diff
@@ -195,6 +235,10 @@ export class Map extends State implements INextable {
       errors.push('maxConcurrency has to be a positive integer');
     }
 
+    if (this.parameters !== undefined && this.itemSelector !== undefined) {
+      errors.push('parameters has been deprecated and only itemSelector should be provided');
+    }
```

Comment:
This should be a warning, not an error. `Annotations.of(this).add('parameters has been deprecated, use itemSelector should be provided`.

Reply:
The error here is only when both parameters and itemSelector are provided in the same Map State.  Parameters should still be supported.  ItemSelector should also be supported.  But supplying both should not be supported and should be an error.